### ISSUE TITLE
High training accuracy, low testing accuracy

### ISSUE THREAD

**Issue by 345696827** [Aminullah6264/BidirectionalLSTM#5]
![image](https://github.com/aminullah6264/Pytorch-Action-Recognition/assets/61975238/f0d64f16-28ec-4507-9704-a8f2fe6a9219)
"Dear author, thank you very much for your open source work. I ran your code and found that the training accuracy is very high, but the testing accuracy is quite low. Is there any issue with the code?"

**Comment by aminullah6264** [comment]
The initial implementation involved features extraction using Caffe and training the LSTM component in TensorFlow. However, for this PyTorch implementation, additional effort is needed for tuning the hyperparameters. I would recommend to try more recent baselines.
